asreviewcontrib/datatools/entrypoint.py

```diff
@@ -59,6 +60,43 @@ def execute(self, argv):
                     type=str,
                     help="Persistent identifier used for deduplication. Default: doi.",
                 )
+                dedup_parser.add_argument(
+                    "--drop_similar",
+                    action='store_true',
+                    help="Drop similar records.",
+                )
+                dedup_parser.add_argument(
```

Comment:
Maybe call this `threshold`?

Reply:
The comments came in the wrong place I think. You mean the `similarity` right?